backend/src/firebase/config.ts

```diff
@@ -1,11 +1,8 @@
-import * as admin from "firebase-admin";
-import { readFileSync } from "fs";
-import { resolve } from "path";
+import * as admin from 'firebase-admin';
+import { readFileSync } from 'fs';
+import { resolve } from 'path';
 
-const ADMIN_FIRE_CONFIG = readFileSync(
-  resolve("./src/config/firebase-admin-sdk.json"),
-  "utf-8"
-);
+const ADMIN_FIRE_CONFIG = readFileSync(resolve('./src/config/firebase-admin-sdk.json'), 'utf-8');
```

Comment:
I cannot see this file anywhere in the repo, do not you store it only locally? I think we need to push it to git as well otherwise it will crash here, have you discussed this with @YurijPaharev? Not sure what is the deployment process here

Reply:
we shared in slack, I AGREED with Yurii to place it in .gitignore file

Reply:
ok I see, @YurijPaharev Are we going to push the` firebase-admin-sdk.json` to the hosting service? Otherwise, I think it won't work once it's deployed, right?

Reply:
yes, @BrantalikP, you are right, we would need this file to be pushed to the deployment platform